src/libraries/Common/src/System/Net/Security/SslClientAuthenticationOptionsExtensions.cs

```diff
@@ -14,6 +14,7 @@ internal static class SslClientAuthenticationOptionsExtensions
     {
         public static SslClientAuthenticationOptions ShallowClone(this SslClientAuthenticationOptions options)
         {
+#pragma warning disable CA1416 // Validate platform compatibility
             var clone = new SslClientAuthenticationOptions()
```

Comment:
Assembly System.Net.Security is not supported on browser, and the API called in multi-targeted context, suppressing for now

Reply:
> suppressing for now

yes, suppressing is not a solution, I meant i can't find correct handling so suppressing for now and seeking suggestions. Any other normal ways of handling this (instantiate it conditionally or Assert, add unsupported attribute) doesn' make sense. Possibly the called API should be supported on browser or this type should be excluded from the browser context

Reply:
This is not referenced from browser context, so excluded from browser build